src/Widgets/HeaderBar.vala

```diff
@@ -107,17 +105,17 @@ namespace Quilter.Widgets {
                 _("Find…")
             );
 
-            if (settings.searchbar == false) {
+            if (gsettings.get_boolean("searchbar") == false) {
                 search_button.set_active (false);
             } else {
-                search_button.set_active (settings.searchbar);
+                search_button.set_active (gsettings.get_boolean("searchbar"));
             }
 
             search_button.toggled.connect (() => {
     			if (search_button.active) {
```

Comment:
If you're just comparing against `search_button.active` why not bind that property directly to the setting?

Reply:
Going to do so outside this PR.